ScrapPackedLibrary/DataTypes/PackedMetaData.cs

```diff
@@ -1,36 +1,37 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Text;
 
 namespace ch.romibi.Scrap.Packed.PackerLib.DataTypes
 {
     public class PackedMetaData
     {
-        private const UInt32 DATA_LENGTH_STATIC = 12; // 4 bytes each: "PFBK", version (all 0s), number of files
-
-        public const string fileHeader = "BFPK";
-        public UInt32 packedVersion { get; set; } // always 0 (not sure if really a version number)
-        public List<PackedFileIndexData> fileList { get; set; }
-        public Dictionary<string, PackedFileIndexData> fileByPath { get; set; }
+        // Fields
+        public const string FileHeader = "BFPK";
+        public uint PackedVersion { get; set; } // always 0 (not sure if really a version number)
+        public List<PackedFileIndexData> FileList { get; set; }
+        public Dictionary<string, PackedFileIndexData> FileByPath { get; set; }
 
+        // Methods
         public void RecalcFileOffsets()
         {
-            UInt32 currentOffset = CalculateFirstFileOffset();
-            foreach (var file in fileList)
-            {
+            uint currentOffset = CalculateFirstFileOffset();
+            foreach (PackedFileIndexData file in FileList) {

```

Comment:
any specific reason to not use `var`?

Reply:
Best practise. 
But I didn't mean to do it. This is default configuration of code-style in VS2022 (I think, or may be I changed it by accident) and IDE changed all `var` to types when I clicked on "code cleaner" button (not sure how it called in English)